client/components/like-button/button.jsx

```diff
@@ -83,8 +83,14 @@ var LikeButton = React.createClass( {
 
 		containerClasses = classnames( containerClasses );
 
+		if ( likeCount === 0 && ! this.props.showCount ) {
+			likeCount = '';
+		} else {
+			likeCount = likeCount + ' ';
```

Comment:
it is. I'd rather not mess with padding / margins here because the elements are `inline`.